<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogColors</class>
 <widget class="QDialog" name="DialogColors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Plain Text Foreground</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonFG</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFG">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&amp;Text Area Background</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonBG</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBG">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&amp;Selection Foreground</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonSelectionFG</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSelectionFG">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Selection &amp;Background</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonSelectionBG</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSelectionBG">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&amp;Matching (..) Foreground</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonMatchFG</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMatchFG">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Matc&amp;hing (..) Background</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonMatchBG</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMatchBG">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&amp;Line Numbers Foreground</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonLineNumbersFG</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLineNumbersFG">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>L&amp;ine Numbers Background</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonLineNumbersBG</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLineNumbersBG">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&amp;Cursor Color</string>
       </property>
       <property name="buddy">
        <cstring>pushButtonCursor</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCursor">
       <property name="styleSheet">
        <string notr="true">text-align:left;padding:4px 4px 4px 20px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>NOTE: Foreground colors only apply when syntax highlighting is DISABLED.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogColors</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
